packages/trace-viewer/src/ui/callTab.tsx

```diff
@@ -44,8 +44,8 @@ export const CallTab: React.FunctionComponent<{
     <div className='call-tab'>
       <div className='call-line'>{action.title}</div>
       <div className='call-section'>Time</div>
-      <DateTimeCallLine name='start:' value={startTime} />
-      <DateTimeCallLine name='duration:' value={renderDuration(action)} />
+      {renderProperty({ name: 'start', type: 'literal', text: startTime })}
```

Comment:
Do you mean to say 'time' or 'ticks'? Not quite sure what literal conveys in this context.